specification.markdown

```diff
@@ -556,13 +556,13 @@ minOccurs: 1
 
 maxOccurs: 1
 
-range:  1
+range:  not 0
```

Comment:
The text already says 

> `EBMLVersion` SHOULD be 1.

It's a soft requirement because one day value 2 may be valid.